Caltech-IPAC/firefly
# Firefly-1189: click to search entire table

#### [Firefly-1189](https://jira.ipac.caltech.edu/browse/FIREFLY-1189):
- added the ability to allow user to search whole table from the click to search (binocular) tool dropdown 

#### Testing:  
  - https://fireflydev.ipac.caltech.edu/firefly-1189-click-action/firefly
  - Go to TAP panel, search for any number of tables (or upload tbl files) 
    - from any of these tables, click on the '**click to search**' (binocular) tool
    - select '**Search Entire Table**' (this can be reworded) 
    - this should take you to the TAP panel with your table loaded into the upload panel. You will be able to see table name, rows, columns selected etc. If this is the first time you clicked on '**Search Entire Table**' for any of the tables, you'll also see the '**upload table/load existing table**' dialog popup. 
       - this part is up for discussion. I am looking into possible solutions for this issue, one option being we don't bring up this dialog by default at all, if the user wants to use it they need to click on the "**Add...**" or "**Change...**" button next to the '**Upload Table:**' label. Any suggestions welcome.
